masterlist.yaml

```diff
@@ -1245,15 +1260,6 @@ plugins:
   - name: 'TrueStormsSE.esp'
     tag:
       - Sound
-  - name: 'DLCIntegration.esp'
```

Comment:
Please try to avoid moving entries around in future, it's confusing.

Reply:
I generally try to avoid it, it made sense in this case due to the type of mod and the size of the mod entry at the time.

Reply:
> Personally I'd like to sort as many of them as possible into default groups.

The order of entries in the masterlist is meaningless and always will be, so it's just a cosmetic thing.